packages/react-mutation-mapper/src/util/DataFetcherUtils.ts

```diff
@@ -15,7 +15,7 @@ import { OncoKbAPI, OncoKBInfo } from 'oncokb-ts-api-client';
 import _ from 'lodash';
 
 export const DEFAULT_MUTATION_ALIGNER_URL_TEMPLATE =
-    'https://www.cbioportal.org/getMutationAligner.json?pfamAccession=<%= pfamDomainId %>';
+    'https://www.cbioportal.org/proxy/mutationaligner.org/api/domains/<%= pfamDomainId %>?metadata=true';
```

Comment:
@onursumer actually would be good to use the specific instance's proxy (i.e. genie uses genie.cbioportal.org/proxy). Can use this function: https://github.com/cBioPortal/cbioportal-frontend/blob/b0b2d3852ccafdc3f5fa41ab611bd4de26a2c129/src/shared/api/urls.ts#L191-L203

Reply:
To be able to use mutation aligner outside the cbioportal context, we should still keep the default proxy URL. However, each portal instance should use its own proxy URL. PR has been updated accordingly.